try:
    a = int('')
except ValueError as e:
    print('Error:', e)
